expressEngine/orderItems/service/orderItems.js

```diff
@@ -0,0 +1,30 @@
+// import service libraries
+import { BaseService, DbService } from '../../lib/service/index';
+
+// import collections
+import { orderItem } from '../../model/index';
+// import messages
+//import cms from '../../cms/orderItems/index';
+
+class Service extends BaseService {
+  registerorderItems = async (data) => {
+    try {
+      const requiredFields = ["orderId", "productId", "unitPrice", "qunatity"];
+      
+      this.validateRequired(data, requiredFields);
+      
+      const orderItems = await DbService.create(orderItem, {
+        orderId: data.orderId,
+        productId: data.productId,
+        unitPrice: data.unitPrice,
+        qunatity: data.qunatity,
```

Comment:
Rather than assigning the object data value, deconstruct the data object and use the property value shorthand